test/test_hegv.cc

```diff
@@ -208,6 +221,14 @@ void test_hegv_work(Params& params, bool run)
 
         // compute and save timing/performance
         params.time() = time;
+        if (timer_level >= 2) {
+            params.time2() = slate::timers[ "hegv::potrf" ];
+            params.time3() = slate::timers[ "hegv::hegst" ];
+            params.time4() = slate::timers[ "hegv::heev" ];
```

Comment:
`hegv` should inherit all the `heev` timers, that is, add these:
```
            params.time2() = slate::timers[ "heev::he2hb" ];
            params.time3() = slate::timers[ "heev::hb2st" ];
            params.time4() = slate::timers[ "heev::stev" ];
            params.time5() = slate::timers[ "heev::unmtr_hb2st" ];
            params.time6() = slate::timers[ "heev::unmtr_he2hb" ];
```
(Copied from test_heev.cc; change timer numbers.)

Reply:
Question, would you not need to add to the code section which names the output column names as well?
```
    if (timer_level >= 2) {
        params.time2();
        params.time3();
        params.time4();
        params.time5();
        params.time6();
        params.time2.name( "potrf (s)" );
        params.time3.name( "hegst (s)" );
        params.time4.name( "heev (s)" );
        params.time5.name( "trsm (s)" );
        params.time6.name( "trmm (s)" );
    }
```
I get that the internal timers will run within `hegv` since we call `heev`, but we aren't using the `test_heev.cc` file and instead using `test_hegv.cc` so the column output would follow what I have above.  Which I can add onto, but want to make sure this thought is correct.